src/pages/ExpiringResults/index.js

```diff
@@ -0,0 +1,204 @@
+import React, { Component } from 'react';
+import { connect } from 'dva';
+import {
+  Grid,
+  GridItem,
+  Card,
+  TextContent,
+  Text,
+  TextVariants,
+  Button,
+  Progress,
+  ProgressSize,
+  ProgressMeasureLocation,
+  ProgressVariant,
+} from '@patternfly/react-core';
+import { Icon } from 'antd';
+import moment from 'moment';
+import { EllipsisVIcon } from '@patternfly/react-icons';
+import { resultData, expirationLimit } from '../../../mock/overview';
+import Table from '@/components/Table';
+import styles from './index.less';
+
+@connect(({ user }) => ({
+  user: user.user,
+  favoriteControllers: user.favoriteControllers,
+}))
+class ExpiringResults extends Component {
+  showDrowpdown = id => {
+    const dropdownElem = document.getElementById(id);
+    if (dropdownElem.style.display === 'none') {
+      dropdownElem.style.display = 'block';
+    } else {
+      dropdownElem.style.display = 'none';
+    }
+  };
+
+  render() {
+    const { favoriteControllers } = this.props;
+    const seenDataColumns = [
+      {
+        title: 'Result',
+        dataIndex: 'result',
+        key: 'result',
+        render: text => (
+          <div>
+            <Button
+              variant="link"
+              isInline
+              style={{ marginBottom: '8px' }}
+              onClick={() => this.navigateToRunResult(text[0])}
+            >
+              {text[0]}
+            </Button>
+            <br />
+            <Text component={TextVariants.p} className={styles.subText}>
+              <span className={styles.label}>{text[1]}</span>
+            </Text>
+          </div>
+        ),
+      },
+      {
+        title: 'End Time',
+        dataIndex: 'end',
+        key: 'end',
+      },
+      {
+        title: 'Scheduled for deletion on',
+        dataIndex: 'deletion',
+        key: 'deletion',
+        render: text => {
+          const deleteDate = moment(new Date(Date.parse(text)));
```

Comment:
I believe you created a utility function to handle generating the date text that we can include a reference to here.

Reply:
That's right. Used the same function now.